Libraries/Components/Button.js

```diff
@@ -154,6 +155,40 @@ type ButtonProps = $ReadOnly<{|
    * For arrow keys, add "ArrowLeft", "ArrowRight", "ArrowUp", "ArrowDown",
    */
   validKeysUp?: ?Array<string>,
+  
+  /**
+   * Specifies whether the view can become key view
+   */
+  canBecomeKeyView?: ?boolean, // TODO(macOS ISS#2323203)
```

Comment:
How will this property differ from the existing `focusable` prop?

Reply:
Hm, maybe `focusable` should internally map to this property, or it's setter sets this property as well?

Reply:
Probably seems fair to map focusable here